docs/en/Community-Articles/2023-01-30-How-To-Contribute-To-a-Open-Source-Software/how-can-I-contribute-to-open-source-projects-on-github.md

```diff
@@ -0,0 +1,105 @@
+# How to Contribute to ABP Framework
+
+## Introduction
+
+In this article I will explain how you can contribute to the open source ABP Framework. You will not only learn about the ABP Framework, but also how to contribute to an open source project, what are the standard rules, some git operations, etc.
+
+## What is Open Source?
+
+Open source software is code designed to be publicly available. Anyone can view, use, modify and distribute the project and code. The fact that the code is open source makes it a natural community and open for improvement. This enables ideas and thoughts to spread rapidly.
+
+## What is ABP Framework?
+
+ABP Framework is a complete infrastructure for building modern web applications following software development best practices and guidelines. ABP Framework is completely free, [open source](https://github.com/abpframework) and community driven. ABP is a modular framework and Application Modules provide pre-built application functionalities.
+
+## Before Contribution
+
+Before making any changes and trying to push them to the target repository we need to create a new [issue](https://github.com/abpframework/abp/issues) if there are no issues with the work. If there is an existing issue, you can proceed through this issue. This way, no other developer will work on the same issue and your PR will have a better chance to be accepted.
+
+Previous ABP Community Talk on this topic can be found here  https://www.youtube.com/watch?v=Wz4Z-O-YoPg
```

Comment:
Use markdown links. For example, "Previous ABP Community Talk on this topic can be found [here](https://www.youtube.com/watch?v=Wz4Z-O-YoPg)."

See more: https://www.markdownguide.org/basic-syntax/#links